lib/tasks/rake_arguments.rb

```diff
@@ -0,0 +1,9 @@
+class RakeArguments
+  def self.task_options
```

Comment:
The filename... this has to do with the task options... so they aren't arguments to rake, they are task arguments... or task options... Perhaps my first choice of RakeArguments is to blame.  I think this module is more like TaskArguments, though generally that could be `Arguable`. Pun intended, but only half joking.